privacyidea/lib/counter.py

```diff
@@ -72,3 +72,17 @@ def reset(counter_name):
         counter = EventCounter(counter_name, 0)
         counter.save()
     counter.reset()
+
+
+def read(counter_name):
+    """
+    Read the counter value from the database
+
+    :param counter_name:
+    :return:
```

Comment:
We should add that this returns None if the counter can't be found.

Reply:
Will do so.